### PR TITLE
Avoid printing control characters for displaying assets

### PR DESCRIPTION
**Pull request by nicholasguoalgorand**
## Summary

Asset names can have control characters, which will be displayed to the terminal when run commands such as `goal asset info`. This allows malicious asset names to delete terminal input and / or print arbitrary input.

## Test Plan

Expect test

### CODE REVIEW

File: cmd/goal/asset.go
Comment (tsachiherman) [inline cmd/goal/asset.go:606]: I think that `UnitName` would be as susceptible as `AssetName`, wouldn't it ?

**Review by tsachiherman — COMMENTED** [review]
Could you also review `printAccountInfo` under the same context ?

File: cmd/goal/asset.go
Comment (nicholasguoalgorand) [reply cmd/goal/asset.go:606]: unit name is limited to 8 characters, would this make any difference?

Comment (tsachiherman) [reply cmd/goal/asset.go:606]: if these contain control characters, it might.

**Review by tsachiherman — APPROVED** [review]
Looks good to me, I'll wait for feedback from @jasonpaulos before merging this in.

**Review by jasonpaulos — APPROVED** [review]
Looks like this fixes the issue for `goal account info/list` too.

**Pull request merged**
